### PR TITLE
added navbar

### PR DESCRIPTION
**Pull request by farhan2077**
#### Navbar in PSD:

***

### CODE REVIEW

File: src/components/Navbar.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        <Menu
+          onClick={this.handleClick}
+          mode="horizontal"
+          className="navbar-bottom"
+        >
+          <Menu.Item key="Home" icon={<HomeOutlined />}>
+            Home
+          </Menu.Item>
+
+          <SubMenu key="Shop" icon={<ShopOutlined />} title="Shop">
```
Comment (alamiiin) [inline src/components/Navbar.js:49]: Can you render the sub-menus using a list? So we can render it dynamically later on?

Comment (farhan2077) [reply src/components/Navbar.js:49]: In ant design, the sub-menus are like the customized version of `<li>`. So, yes, we can use `<li>` to render sub-menus. But then those `<li>` items must be styled with `css`.
This is what happens when I use `<li>`to render sub-menus without using any `css`.


Screenshot (83)

As for dynamic rendering, I think sub-menus can be dynamically rendered.

Comment (alamiiin) [reply src/components/Navbar.js:49]: Can't you just loop over `Menu.Item` ? Why you need to use raw `li`?

Comment (farhan2077) [reply src/components/Navbar.js:49]: I am not understanding clearly that why do we need to loop over `Menu.item` to render those dynamically. But as for dynamically rendering different navbar based on whether the user is logged in or not, I can do that.

**Review by alamiiin — APPROVED** [review]

**Pull request merged**